core/src/replay_stage.rs

```diff
@@ -179,58 +189,55 @@ impl ReplayStage {
         bank_forks: &Arc<RwLock<BankForks>>,
         poh_recorder: &Arc<Mutex<PohRecorder>>,
         cluster_info: &Arc<RwLock<ClusterInfo>>,
+        blocktree: &Blocktree,
+        ticks_per_slot: u64,
```

Comment:
looks like you need to calculate a slot, why not do this above?

Reply:
donezo